packages/dds/cell/src/cell.ts

```diff
@@ -158,23 +219,40 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>> impl
         return this.data === undefined;
     }
 
+    /**
+     * {@inheritDoc ISharedCell.getAttribution}
+     * @alpha
+     */
+    public getAttribution(): AttributionKey | undefined {
+        return this.attribution;
+    }
+
+    /**
+     * {@inheritDoc ISharedCell.setAttribution}
+     * @alpha
+     */
+    private setAttribution(message: ISequencedDocumentMessage): void {
+        this.attribution = { type: "op", seq: message.sequenceNumber };
+    }
+
     /**
      * Creates a summary for the Cell.
      *
      * @returns The summary of the current state of the Cell.
      */
     protected summarizeCore(serializer: IFluidSerializer): ISummaryTreeWithStats {
-        const content: ICellValue = { value: this.data };
+        const content: ICellContent = { value: this.data, attribution: this.attribution };
         return createSingleBlobSummary(snapshotFileName, serializer.stringify(content, this.handle));
     }
 
     /**
      * {@inheritDoc @fluidframework/shared-object-base#SharedObject.loadCore}
      */
     protected async loadCore(storage: IChannelStorageService): Promise<void> {
-        const content = await readAndParse<ICellValue>(storage, snapshotFileName);
+        const content = await readAndParse<ICellContent>(storage, snapshotFileName);
 
-        this.data = this.decode(content);
+        this.data = this.decode({ value: content.value });
```

Comment:
this looks like it breaks back-compat for existing serialized SharedCells. We need to make sure we preserve that, and since it seems like tests haven't failed, we need to add format compatability tests.

Reply:
oh wait, nevermind this format is pre-existing. sorry for the trouble. (it was already wrapped in `value`, just didn't have `attribution`)